Classes/New Issue/IssueTemplates.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  IssueTemplates.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+struct IssueTemplate {
+    let title:String
+    let template:String
+}
+
+enum IssueTemplateHelper {
+
+    static private func matchesForRegexInText(
+        regex: String!,
+        text: String!) -> [String] {
+
+        do {
+            let regex = try NSRegularExpression(pattern: regex, options: [])
```

Comment:
FYI you can omit the `options` parameter as that has a default expression of `[]`.